### PR TITLE
Further reformatting of variable documentation

### PR DESCRIPTION
**Pull request by pajlada**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

Left is master, right is this PR
image

### CODE REVIEW

File: docs/variables.md [hunk 1]
```diff
@@ -17,22 +17,25 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
-`slice` - slice the string like a python3 string - see https://www.digitalocean.com/community/tutorials/how-to-index-and-slice-strings-in-python-3 (NOTE: We don't support what they call 'stride')\
-Math - add, subtract, multiply, divide:
+#### Uncategorized
+
+`strftime` - allows you to format raw time strings - Help: https://strftime.org  
+`lower` - converts the output of the variable to lowercase  
+`upper` - converts the output of the variable to uppercase  
+`title` - converts the first letter in each word to upper case  
+`capitalize` - converts the first letter of the first word to upper case  
+`swapcase` - swaps all upper case to lower case and vice versa
```
Comment (alazymeme) [inline docs/variables.md:27]: Are we able to categorize these as "Case-related items"?

File: docs/variables.md [hunk 2]
```diff
@@ -17,22 +17,25 @@ Examples for valid substitutions: `$(user;1:points)` - get the user with the log
...
+
+`strftime` - allows you to format raw time strings - Help: https://strftime.org  
+`lower` - converts the output of the variable to lowercase  
+`upper` - converts the output of the variable to uppercase  
+`title` - converts the first letter in each word to upper case  
+`capitalize` - converts the first letter of the first word to upper case  
+`swapcase` - swaps all upper case to lower case and vice versa  
+`time_since_minutes` - outputs the time since a certain input time in minutes  
+`time_since` - outputs the time since a certain input time  
+`time_since_dt` - outputs the date since a certain input date
```
Comment (alazymeme) [inline docs/variables.md:30]: Can we categorize these into a "Time Since" category?

File: docs/variables.md (hunk 1 above)
Comment (pajlada) [reply docs/variables.md:27]: we could in another pr yes

File: docs/variables.md (hunk 2 above)
Comment (pajlada) [reply docs/variables.md:30]: we could in another pr yes

File: docs/variables.md (hunk 1 above)
Comment (pajlada) [reply docs/variables.md:27]: or you could pr my pr

**Review by RAnders00 — APPROVED** [review]
👍🏻

**Pull request merged**